rotkehlchen/db/upgrades/v34_v35.py

```diff
@@ -50,6 +51,13 @@ def _rename_assets_identifiers(cursor: 'DBCursor') -> None:
     cursor.executemany('UPDATE OR IGNORE assets SET identifier=? WHERE identifier=?', sqlite_tuples)  # noqa: E501
 
 
+def _add_blockchain_column_web3_nodes(cursor: 'DBCursor') -> None:
+    cursor.execute("ALTER TABLE web3_nodes RENAME TO web3_nodes_old")
+    cursor.execute(DB_CREATE_WEB3_NODES)
```

Comment:
I understand the urge to re-use the code and in every other case this would be the right thing to do, except from here.

You can not reuse code in DB upgrades since at any point in the future `DB_CREATE_WEB3_NODES` may be modified, but what this upgrade does should always be the same and stay constant so far as table schemas are concerned.

So here you need to copy paste the code exactly